src/main.rs

```diff
@@ -122,16 +128,24 @@ fn spin_toml_path() -> Result<String> {
         + "/fermyon-cloud-gpu/spin.toml")
 }
 
-fn print_runtime_config(auth_token: String) {
+fn print_runtime_config(url: String, auth_token: String) {
     println!("Add the following configuration to your runtime configuration file.");
-    println!("You'll need replace <URL> with the URL of your fermyon-cloud-gpu Spin app.");
-    println!("You can find the URL in the Cloud dashboard.");
     println!(
         r#"
-[llm_compute]
-type = "http"
-url = "<URL>"
-auth_token = "{auth_token}"
-    "#
+        [llm_compute]
```

Comment:
Maybe my copying and pasting skills are inadequate but might it be good to print this without leading spaces for easy copying?

Reply:
It prints without the the leading spaces?
![image](https://github.com/fermyon/spin-cloud-gpu/assets/25023490/a97e36fb-dc29-42d2-b496-461b42d60709)

Reply:
How odd!  When I copied the to a test harness it printed indented.  But I trust your proper program than my wee test, so ship it and apologies for the false alarm!

Reply:
...but why is the output in your screenshot in a different order from what's in the code?

Reply:
The order is different because it is an older screenshot 😅

Reply:
Oh with the updated code - it is indented - will fix it.

Reply:
![image](https://github.com/fermyon/spin-cloud-gpu/assets/865538/a63575cd-8469-4794-a000-f3d50693b6aa)